deployment/terraform/strings.go

```diff
@@ -247,6 +247,16 @@ net.ipv4.tcp_fin_timeout = 30
 
 # Reuse TIME-WAIT sockets for new outgoing connections.
 net.ipv4.tcp_tw_reuse = 1
+
+# TCP buffer sizes are tuned for 10Gbit/s bandwidth and 0.5ms RTT (as measured intra EC2 cluster).
+# This gives a BDP (bandwidth-delay-product) of 625000 bytes.
```

Comment:
I know this is copied from the app nodes, but it's a good opportunity for me to understand it better. The BDP computation I do understand it:
BDP = bandwidth * delay  = 10Gbit/s * 0.5ms = (10 * 10^9) * (0.5 * 10^-3) = 5000000 bits = 625000 bytes

But how are the other numbers computed? Are they all defined in terms of the BDP?

Reply:
Also, is the RTT the same intra-cluster than between agents and proxy? Should we double-check that?

Reply:
@streamer45 wrote this. I'll defer to him.